cnd/src/seed.rs

```diff
@@ -191,6 +192,18 @@ impl SwapSeed {
     pub fn derive_secret(&self) -> Secret {
         self.sha256_with_seed(&[b"SECRET"]).into()
     }
+
+    /// Used to derive the transient redeem identity for hbit swaps.
+    pub fn derive_transient_redeem_identity(&self) -> SecretKey {
```

Comment:
:question: Shouldn't these functions live in a `hbit` module? It seems to me like the swap seed shouldn't know about protocols.

Reply:
Good call, I like that idea :)